bears/xml2/XCopBear.py

```diff
@@ -0,0 +1,57 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+import re
+import itertools
+
+
+@linter(executable='xcop',
+        use_stdout=True)
+class XCopBear:
+    """
+    This bear can fix formatting errors in your XML file.
+    Check https://github.com/yegor256/xcop for more details.
+    """
+    LANGUAGES = {'XML', 'XSD', 'XSL', 'XHTML'}
+    REQUIREMENTS = {GemRequirement('xcop')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    _output_regex = re.compile(
+        r'.*:(?P<line>\d+):.*(?P<severity>error|warning)\s?: '
+        r'(?P<message>.*)\n.*\n.*')
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return filename,
+
+    def process_output(self, output, filename, file):
+        stdout = output
+        invalid_format = re.compile(r'\x1b\[31m.+?0m', re.S)
+        correct_formating = filename + ' looks good\n'
+
+        if(stdout != correct_formating):
+            """Output of Xcop is colour coded in the terminal to
+               identify the changes suggested by xcop.Green to indicate
+               addition and red to denote subtraction.
```

Comment:
Is it possible to disable color mode? (Most tool have an option for that)

Reply:
and then you don't need to specify `process_output` manually

Reply:
I emailed the maintainer of the repo and also raised an issue ... There was no response

Reply:
Have you looked at `xcop --help`?

Reply:
bump

Reply:
Yes I did also raised an issue with the developer. He said it is something that can be done in the future. No provision as of now

Reply:
link to the issue ?

Submit a PR to have a no-color option added ?

Reply:
https://github.com/yegor256/xcop/issues/19 This is the issue I raised